snp_motion_planning/src/plugins/tasks/tcp_speed_limiter_profile.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <memory>
+#include <string>
+
+static const std::string TCP_SPEED_LIMITER_TASK_NAME = "TCPSpeedLimiterTask";
+
+namespace snp_motion_planning
+{
+struct TCPSpeedLimiterProfile
+{
+  using Ptr = std::shared_ptr<TCPSpeedLimiterProfile>;
+  using ConstPtr = std::shared_ptr<const TCPSpeedLimiterProfile>;
+
+  TCPSpeedLimiterProfile(const double max_speed_ = 0.25, const std::string tcp_ = "tool0")
```

Comment:
Let's remove the default values here. I don't think it makes sense to provide them, since they will likely be different for each application